### PR TITLE
login page and logic

### PR DESCRIPTION
**Pull request by alex-morikado**
its a new page that redirects to index.html once a login is successful

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```
Comment (vdhalex) [inline login.html:1]: Why is this in login.html instead of index.html?

File: login.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+String.prototype.hash = function() {
+    var hash = 0;
+    for (var i = 0; i < this.length; ++i) {
+        hash = ((hash << 5) - hash) + this.charCodeAt(i) | 0;
+    }
+    return hash;
+};
+
```
Comment (vdhalex) [inline login.js:8]: you're so extra how are we supposed to catfish passwords

File: login.html (hunk 1 above)
Comment (alex-morikado) [reply login.html:1]: great question

Comment (alex-morikado) [reply login.html:1]: didnt want to create merge conflicts lol

Comment (alex-morikado) [reply login.html:1]: we can integrate once youre done

Comment (vdhalex) [reply login.html:1]: fair we can merge them after

**Review by vdhalex — APPROVED** [review]

**Pull request merged**